### ISSUE TITLE
Jinja whitespace control ignored

### ISSUE THREAD

**Issue by OliElli** [dmulyalin/salt-nornir#21]
Looks like whitespace control in Jinja templates is ignored. 

```
[olielli@master ~]$ sudo salt nornirproxy nr.cfg_gen "{% for i in ['1', '2', '3'] -%}\n{{ i }}\n{% endfor -%}"
nornirproxy:
    ----------
    switch1:
        ----------
        salt_cfg_gen:

            1

            2

            3
```
These interim newlines shouldn’t be there, thats what the `-%}` statement should address.

Preferrably there should be a way to include Jinja config like `lstrip_blocks=true` and `trim_blocks=true` which is already configured on the master.

**Comment by dmulyalin** [comment]
Thank you for raising this issue.

Was testing this one and came up with solution to honour lstrip_blocks=true and trim_blocks=true config in master config file or in proxy minion pillar file, will include fix in next release.

**Comment by dmulyalin** [comment]
Should be addressed in 0.15.0 release

**Comment by OliElli** [comment]
Confirmed that the master jinja config is now honoured, nice!
